------------------------------------------------------------------------
r570840 | mueller | 2006-08-07 22:26:20 +0200 (Mon, 07 Aug 2006) | 3 lines
Changed paths:
   M /branches/KDE/3.5/kdebase/kicker/taskbar/taskbar.cpp

fix stupid typo
BUG: 128552

------------------------------------------------------------------------
Index: kicker/taskbar/taskbar.cpp
===================================================================
--- kicker/taskbar/taskbar.cpp	(revision 570839)
+++ kicker/taskbar/taskbar.cpp	(revision 570840)
@@ -177,14 +177,14 @@ QSize TaskBar::sizeHint( KPanelExtension
 
         if (containerCount() == 0)
         {
-            actualMax = maxSize.height();
+            actualMax = minButtonHeight;
         }
 
         if (actualMax > maxSize.height())
         {
             return maxSize;
         }
-        return QSize( maxSize.width(), maxSize.height() );
+        return QSize( maxSize.width(), actualMax );
     }
     else
     {
